pharus/component_interface.py

```diff
@@ -114,6 +133,15 @@ def restriction(self):
             ]
         )
 
+class InsertComponent(QueryComponent):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def dj_query_route(self):
+        [t.insert1(
```

Comment:
We'll need to put this into a [transaction](https://docs.datajoint.org/python/manipulation/4-Transactions.html) context. Also, for functions that do not return data, it is atypical to include into a list. It is tempting to put them into a list comprehension and I used to do this too. :sweat_smile:. It started becoming difficult to review after some time...